reflect-lint/gradle.properties

```diff
@@ -0,0 +1,3 @@
+POM_NAME=Lint Rules for Dagger Reflect
+POM_ARTIFACT_ID=dagger-reflect-lint
+POM_PACKAGING=jar
```

Comment:
should this be a `jar` or `aar`? or is that not relevant for the pom?

Reply:
By using a `jar` it should theoretically be possible to use it with non-Android projects and the standalone Lint version.

Reply:
It always has to be a jar, even when embedding in aars or depending via the `lintChecks` configuration